### DISCUSSION TITLE
Please add --concise flag to `sf project retrieve preview`

### DISCUSSION THREAD

**Discussion by jclark-dot-org** [forcedotcom/cli#2489] [Ideas]
Back when `sfdx force:source:status` was updated (the "beta" commands), it started including a list of ignored files, which was noisy.  SFDX v7.140.0 [added](https://github.com/forcedotcom/cli/blob/main/releasenotes/sfdx/README.md#71400-march-3-2022) the `--concise` flag to make the output more like the original output, such as suppressing this list of ignored metadata.

Today, `sf force:source:status --concise` still works, but gives the warning, "Warning: We plan to deprecate this command in the future. Try using the "project retrieve/deploy preview" command instead."  However, `sf project retrieve preview` does not support `--concise`, nor any similar flag.  Please add this; it is very helpful to be able to see only relevant changes.

**Comment by shetzel** [comment]
@jclark-dot-org - Thanks for starting the discussion.  To clarify, you're just looking for a way to suppress the forceignored files reported by `preview` and not necessarily the `--concise` flag name right?

**Reply by jclark-dot-org** [reply]
@shetzel Correct.  I'm basically looking to get the same output from `project retrieve preview` as I can get today from `force:source:status --concise`.  I find I need "just the differences" much more frequently than the full rundown.

**Comment by shetzel** [comment]
This should now be implemented with https://github.com/salesforcecli/plugin-deploy-retrieve/pull/831
You can `sf plugins:install @salesforce/plugin-deploy-retrieve@latest` to try it out.  It will be in tonight's nightly CLI release, or the CLI release candidate 11/13/2023, or the production CLI on 11/20/2023
